alarms/dojot.docker.ContainerError.xml

```diff
@@ -0,0 +1,11 @@
+<alarm-metamodel>
+  <domain>ContainerError</domain>
+  <namespace>dojot.docker</namespace>
+  <primary-subject>
+    <primary-field type="alphanumeric">container</primary-field>
+    <primary-field type="alphanumeric">image</primary-field>
```

Comment:
Apart from the image name (which does contain a version), it might prove useful to log the image SHA number, thus making easier to spot out-of-date or bogus container images